java/fundamentals/intro-to-java/why-should-i-learn-java.md

```diff
@@ -1,59 +1,51 @@
 ---
 author: mihaiberq
 
-levels:
-
-  - beginner
-
 type: normal
 
 category: feature
 
 aspects: 
-
   - introduction
 
-inAlgoPool: false
-
-
 links:
-
   - '[Who is using Java?](https://www.quora.com/Why-is-Java-so-important){website}'
+  - '[Java Roadmap](https://www.oracle.com/java/technologies/java-se-support-roadmap.html){website}'
 
 ---
 
-# Why Should I Learn Java?
+# Why Learn Java?
 
 ---
 ## Content
 
-*Java* is one of the most used programming languages in the world, with millions of developers. It runs on PCs, mobile phones and even Blu-ray players. 
+Here are several reasons to learn Java:
+
+1. It's one of the most used programming languages in the world (with millions of developers).
 
-One of the reasons to learn Java is because the basic concepts are relatively *easy to understand*. And until you get confident enough, there is really no need to care too much for things like memory management or garbage collection, as the JVM (Java Virtual Machine) takes care of them for you.
+2. The **basic concepts are relatively easy to understand**.
 
-Java is also *static typed*. That means the code cannot be built before solving the errors, which saves time in the long run. Even though the stricter environment doesn't necessarily mean a better one, it allows beginners to stick to less overwhelming rules.
+3. It's **static typed**, meaning that **the code can't be built before solving the errors**. Even though the stricter environment doesn't necessarily mean a better one, it allows beginners to stick to less overwhelming rules.
```

Comment:
wondering if we should also say that this kind of behaviour helps new developers catch bugs more easily

Reply:
yeah, that might show a usefulness for those hat are starting out.

Maybe something like:

```suggestion
3. It's **statically typed**, meaning that **the code can't be built before solving the errors**. Even though the stricter environment doesn't necessarily mean a better one, it allows developers to catch bugs early on.
```